airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/ConfigRepositoryTest.java

```diff
@@ -448,4 +452,31 @@ void testHealthCheckFailure() throws SQLException {
     assertFalse(check);
   }
 
+  @Test
+  void testGetAllStreamsForConnection() throws Exception {
+    final UUID connectionId = UUID.randomUUID();
+    final AirbyteStream airbyteStream = new AirbyteStream().withName("stream1").withNamespace("namespace1");
+    final ConfiguredAirbyteStream configuredStream = new ConfiguredAirbyteStream().withStream(airbyteStream);
+    final AirbyteStream airbyteStream2 = new AirbyteStream().withName("stream2");
+    final ConfiguredAirbyteStream configuredStream2 = new ConfiguredAirbyteStream().withStream(airbyteStream2);
+    final ConfiguredAirbyteCatalog configuredCatalog = new ConfiguredAirbyteCatalog().withStreams(List.of(configuredStream, configuredStream2));
+
+    final StandardSync sync = new StandardSync()
+        .withCatalog(configuredCatalog);
+    doReturn(sync)
+        .when(configRepository)
+        .getStandardSync(connectionId);
+
+    final List<StreamDescriptor> result = configRepository.getAllStreamsForConnection(connectionId);
+    assertEquals(2, result.size());
```

Comment:
Nit: I don't want to block this review for this but this and this is not applicable to this review because it is already using the jupiter assertions.
AssertJ is makes would make it easier to understand such a test and would give you a meaningful diff. It would look like:
```
Assertions.assertThat(result).hasSize(2)
// create the expectedStreamDescriptor here
Assertions.containsOnly(expectedStreamDescriptor, expectedStreamDescriptor2)
```